conf/application.conf

```diff
@@ -179,6 +179,7 @@ feature-switch {
   enable-opt-out = false
   enable-income-sources-new-journey = false
   enable-time-machine = false
+  enable-user-session-API = true
```

Comment:
can we use different name for this FS? something more expressive like, enable-session-data-storage or similar

Reply:
ya sure, this was just the one originally on the ticket